backup-zip.sh

```diff
@@ -1,8 +1,8 @@
 #!/bin/bash
 #set -x
 # MySQL bağlantı bilgileri
-DB_USER="root"
-DB_PASSWORD="my-secret=pw"
```

Comment:
Parametre girilmediğinde script hata verecektir. Lütfen parametre girilmediğinde "parametre girilmedi" diye scripte uyarı verdirebilir miyiz?

Reply:
Fixed, can you check again?